MathMagician/Numbers/PrimeNumber.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MathMagician.Numbers
+{
+    public class PrimeNumber : NaturalNumber
+    {
+        public PrimeNumber()
+        {
+            First = 2;
+        }
+
+        public override int GetNext(int current)
+        {
+            int next = current + 1;
+            bool IsNotFound = true;
+          
+            while(IsNotFound)
```

Comment:
I like the naming of this variable. Makes the loop condition easy to read! :smile: